### ISSUE TITLE
Link doesn't work on Windows stable MSVC

### ISSUE THREAD

**Issue by PsychoPflanze** [deltaphc/raylib-rs#17]
When i follow the installation steps this happens:

`
LINK : warning LNK4098: defaultlib 'MSVCRTD' conflicts with use of oth er libs; use /NODEFAULTLIB:library
libraylib_sys-8c3123562ecb3094.rlib(win32_window.obj) : error LNK2019:  unresolved external symbol __imp_DragQueryFileW referenced in function windowPr oc                                                                                         libraylib_sys-8c3123562ecb3094.rlib(win32_window.obj) : error LNK2019:  unresolved external symbol __imp_DragQueryPoint referenced in function windowPr oc                                                                                         libraylib_sys-8c3123562ecb3094.rlib(win32_window.obj) : error LNK2019:  unresolved external symbol __imp_DragFinish referenced in function windowProc             libraylib_sys-8c3123562ecb3094.rlib(win32_window.obj) : error LNK2019:  unresolved external symbol __imp_DragAcceptFiles referenced in function createN ativeWindow                                                                                C:\Development\rust\raylib-test\target\debug\deps\raylib_test-06c6f1ea 19c27695.exe : fatal error LNK1120: 4 unresolved externals
`

I followed the installation steps and put the raylib.lib in the correct folder

**Comment by nunocp** [comment]
I'm getting similar errors using GNU toolchain (stable-x86_64-pc-windows-gnu), so I can't use raylib-rs at all!

Did you find any solution?

```
= note: W:\Trabalhos\Games_e_Software\God of Monsters\gom\target\debug\deps\libraylib_sys-32a1993cfc0d7dc2.rlib(win32_window.c.obj): In function `windowProc':
          C:/projects/raylib/src/external/glfw/src/win32_window.c:1067: undefined reference to `__imp_DragQueryFileW'
          C:/projects/raylib/src/external/glfw/src/win32_window.c:1071: undefined reference to `__imp_DragQueryPoint'
          C:/projects/raylib/src/external/glfw/src/win32_window.c:1076: undefined reference to `__imp_DragQueryFileW'
          C:/projects/raylib/src/external/glfw/src/win32_window.c:1079: undefined reference to `__imp_DragQueryFileW'
          C:/projects/raylib/src/external/glfw/src/win32_window.c:1091: undefined reference to `__imp_DragFinish'
          W:\Trabalhos\Games_e_Software\God of Monsters\gom\target\debug\deps\libraylib_sys-32a1993cfc0d7dc2.rlib(win32_window.c.obj): In function `createNativeWindow':
          C:/projects/raylib/src/external/glfw/src/win32_window.c:1171: undefined reference to `__imp_DragAcceptFiles'
```

**Comment by nunocp** [comment]
Found out that my problem was already **SOLVED HERE**: #13 
It seems that the root of yours is the same as mine.

As I'm new to Rust and Git, I don't know how to swap/change/merge @borkin8r's branch into my Cargo repository. So I changed the file manually:

`[path to your cargo directory] \registry\src\github.com-[########]\raylib-sys-0.9.2\build.rs`

**Comment by Dacode45** [comment]
Try again with latest 0.11
